examples/tut-10/App.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ * @flow strict-local
+ */
+
+import React from 'react';
+import AppViews from './src/AppViews';
+import DeployerViews from './src/DeployerViews';
+import AttacherViews from './src/AttacherViews';
+import { renderView } from './src/render';
+import * as backend from './build/index.main.mjs';
+import { loadStdlib } from '@reach-sh/stdlib';
+import { Platform } from 'react-native';
+
+const reach = loadStdlib(process.env);
+
+const handToInt = { ROCK: 0, PAPER: 1, SCISSORS: 2 };
+const intToOutcome = ['Bob wins!', 'Draw!', 'Alice wins!'];
+const { standardUnit } = reach;
+const defaults = { defaultFundAmt: '10', defaultWager: '0.005', standardUnit };
+
+class App extends React.Component {
```

Comment:
How come we're using class-based components instead of functional components?

Reply:
No idea. @DanBurton wrote it :)

Reply:
Ha, I see. I was just curious: the syntax for functional components is much cleaner.